Lib/multiprocessing/pool.py

```diff
@@ -264,7 +282,10 @@ def _maintain_pool(ctx, Process, processes, pool, inqueue, outqueue,
         """Clean up any exited workers and start replacements for them.
         """
         if Pool._join_exited_workers(pool):
-            Pool._repopulate_pool_static(ctx, Process, processes, pool,
+            Pool.
+            
```

Comment:
Thanks for the PR. There seems to white spaces in this line and causes `make patchcheck` to fail in CI. Please remove them or run `make patchcheck` locally from the source directory that will fix this for you automatically.

Relevant make patchcheck output : 

```
Getting base branch for PR ... upstream/master
Getting the list of files that have been added/changed ... 5 files
Fixing Python file whitespace ... 1 file:
  Lib/multiprocessing/pool.py
Fixing C file whitespace ... 0 files
Fixing docs whitespace ... 0 files
Docs modified ... yes
Misc/ACKS updated ... NO
Misc/NEWS.d updated with `blurb` ... NO
configure regenerated ... not needed
pyconfig.h.in regenerated ... not needed
```